pom.xml

```diff
@@ -254,6 +254,39 @@
       <version>test-1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.Emibergo02</groupId>
+      <artifactId>RedisChat</artifactId>
+      <version>main-d1a2d70e05-1</version>
+      <scope>provided</scope>
+      <optional>true</optional>
+      <exclusions>
+        <exclusion>
+          <groupId>xyz.xenondevs.invui</groupId>
+          <artifactId>invui-core</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>xyz.xenondevs.invui</groupId>
+          <artifactId>inventory-access-r5</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>xyz.xenondevs.invui</groupId>
+          <artifactId>inventory-access-r7</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>xyz.xenondevs.invui</groupId>
+          <artifactId>inventory-access-r9</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>xyz.xenondevs.invui</groupId>
+          <artifactId>inventory-access-r13</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>xyz.xenondevs.invui</groupId>
+          <artifactId>inventory-access-r14</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
```

Comment:
I wish an explanation for all this exclusions

Reply:
They were not resolvable due to the our mirror setup. Added a mirror for the transitive repository to our nexus.